packages/gcf-utils/test/integration/gcf-bootstrapper-integration.ts

```diff
@@ -55,5 +56,23 @@ describe('GCFBootstrapper Integration', () => {
         payload: 'baz',
       });
     });
+
+    it('provides github with logging plugin', async () => {
+      const pb = await bootstrapper.loadProbot((app: Application) => {
+        app.on('foo', async context => {
+          assert(
+            (context.github as GitHubAPI & {
+              loggingOctokitPluginVersion: string;
+            }).loggingOctokitPluginVersion === '1.0.0'
+          );
+        });
+      });
```

Comment:
We probably want to assert that this block is actually hit.

Reply:
Yes I'm actually still working on getting the integration tests to run -- running into some more access issues. Will add the assert once I can get them running properly.